integration-testing/rnode_testing/network.py

```diff
@@ -63,12 +63,3 @@ def wait_for_converged_network(timeout, network, peer_connections):
         wait_for(has_peers(node, peer_connections),
                  timeout,
                  "The network did NOT converge. Check container logs for issues. One or more containers might have failed to start or connect.")
-
-
-def wait_for_approved_block_received(network, node_startup_timeout):
-    for peer in network.peers:
-        wait_for(
-            approved_block_received(peer),
-            node_startup_timeout,
-            "Peer {} did not receive the approved block",
-        )
```

Comment:
Is this on purpose?  This function appears to be in use.

Reply:
Yes. I did it on purpose. It is a duplicate func in the same file. Check [line 43](https://github.com/rchain/rchain/pull/1924/files/79606ee3db92f425cb56024cbbd96d3b1edd5580#diff-0c0c098d2a85cca40c722f2cbab6931aR43) in the same file.

The mypy test finds it.

Reply:
👍